pkg/crc/network/proxy.go

```diff
@@ -96,6 +97,16 @@ func getProxyFromEnv(proxyScheme string) string {
 	return p
 }
 
+func setEnv(key, value string) {
+	before := os.Getenv(key)
+	if before != value {
+		logging.Warnf("Overriding environment variable %s: previously %s, now %s", key, before, value)
```

Comment:
`logging.Debugf` should be enough? This looks a bit weird when `before` is empty, could be wrapped between `[%s]` maybe, or special cased.